### ISSUE TITLE
Conflict between EEGLAB and LIMO on session in design matrix

### ISSUE THREAD

**Issue by arnodelorme** [LIMO-EEG-Toolbox/limo_tools#171]
When EEGLAB declare that it wants to compare sessions in the STUDY.design, instead of being ignored, it is included in the LIMO design.

**Comment by CPernet** [comment]
It is not ignored - if you have two seesions, it creates to sessions, run the film for each one, and compute between sessions contrasts - the reason to do this is because the variance between session is not the same as if LIMO was doing one design. ++- not a bug, a feature.

**Comment by arnodelorme** [comment]
No, it creates a categorical variable for the session (it also creates different files as it should).

**Comment by CPernet** [comment]
Do you mean, you have a session variable? Yes it does not deal with that -- and I do not want that (session variance needs to be accounted for)

**Comment by arnodelorme** [comment]
Let's change the code to remove the session variable from the STUDY design (temporarily) before computing LIMO files.
